perun-base/src/main/java/cz/metacentrum/perun/rpc/serializer/JsonSerializerGWT.java

```diff
@@ -41,6 +45,57 @@
 	private interface CandidateMixIn {
 	}
 
+	@SuppressWarnings("unused")
+	private interface TaskMixIn {
+		@JsonSerialize
+		long getStartTimeAsLong();
+
+		@JsonIgnore
+		LocalDateTime getStartTime();
+
+		@JsonSerialize
+		long getScheduleAsLong();
```

Comment:
To prevent necessity of renaming all properties on client side (GUI, CLI) you can specify name of JSON property using annotations like ``@JsonProperty(value = "schedule")``. It should be name of original property containing LocalDateTime.